src/Common/src/System/Dynamic/Utils/TypeUtils.cs

```diff
@@ -41,13 +41,9 @@ public static void ValidateType(Type type)
             if (type != typeof(void))
             {
                 // A check to avoid a bunch of reflection (currently not supported) during cctor
```

Comment:
Is this comment still relevant? I can't tell what it is referring to. Reflection should be supported. Is that about checking for void?